src/components/tabs/Tabs.tsx

```diff
@@ -0,0 +1,45 @@
+import 'bulma/css/bulma.css';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { tabs } from '../../data/tabs';
+
+export const Tabs = () => {
+  const { tabId } = useParams();
+  const selectedTabId = tabId || null;
+
+  const getContentById = (id: string | null) => {
+    const foundTodo = tabs.find(tab => tab.id === id);
```

Comment:
The variable name 'foundTodo' is misleading here, because it doesn't deal with any todo, but a tab. It would be better to rename it to 'foundTab' for clarity.